src/core/reactor.cc

```diff
@@ -1195,11 +1195,7 @@ cpu_stall_detector_linux_perf_event::~cpu_stall_detector_linux_perf_event() {
 void
 cpu_stall_detector_linux_perf_event::arm_timer() {
     uint64_t ns = (_threshold * _report_at + _slack) / 1ns;
-    #if __cplusplus >= 202002L
-    if (_enabled && _current_period == ns) [[likely]] {
-    #else
-    if (_enabled && _current_period == ns) {
-    #endif
+    if (__builtin_expect(_enabled && _current_period == ns, 1)) {
```

Comment:
@andy-yx-chen , please merge both commits into one, so I could merge a single fix, not a "fix and a fix for a fix" pair.

Reply:
Okay, I thought squash would work, anyways, I made it to single commit